packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -315,6 +316,8 @@ export const metricsEpic: Epic<AnyAction, AnyAction, any, any> = (action$, state
         subscription_key: payload.subscriptionKey,
       })
 
+      action.payload = redact(action.payload) // use redact function in util file before payload is passed onward
```

Comment:
Redaction should be done at the logging level only, we really shouldnt modify any internal code for it

Reply:
The problem is API keys currently end up in Prometheus because of the URL label.

Reply:
Forget my comments if we implement ☝️